app/controllers/course/assessment/submission/submissions_controller.rb

```diff
@@ -196,15 +211,15 @@ def delete
   def delete_all
     authorize!(:delete_all_submissions, @assessment)
     submission_ids = @assessment.submissions.by_users(course_user_ids).pluck(:id)
-    if !submission_ids.empty?
+    if submission_ids.empty?
+      head :ok
+    else
```

Comment:
Same point as above about changing this to `end`.

Reply:
Noted